Orders.Logic/OrderManager.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+
+namespace Orders.Logic
+{
+    public class OrderManager
+    {
+        private readonly IOrderStore orderStore;
+        public OrderManager(IOrderStore orderStore)
+        {
+            this.orderStore = orderStore;
+        }
+        public void WriteOutSmallOrders()
+        {
+            var orders = orderStore.GetOrders();
+            SmallOrderFilter filter = new SmallOrderFilter(new OrderWriter(),
+           orders);
+            filter.WriteOutFiltrdAndPriceSortedOrders(new OrderWriter());
+        }
+        public void WriteOutLargeOrders()
+        {
+            var orders = orderStore.GetOrders();
+            LargeOrderFilter filter = new LargeOrderFilter(new OrderWriter(),
+           orders);
+            filter.WriteOutFiltrdAndPriceSortedOrders(new OrderWriter());
+        }
+    }
+    public class LargeOrderFilter
+    {
+        private IOrderWriter orderWriter;
+        private List<Order> orders;
+        public LargeOrderFilter(IOrderWriter orderWriter, List<Order> orders)
+        {
```

Comment:
I would establish a relationship between IOrderWriter and IOrderStore. I don't like that you pass in new instances of the OrderWriter class on each Expression resolution. I would move it out this class and into the calling parent class.